clouddriver-docker/src/test/groovy/com/netflix/spinnaker/clouddriver/docker/registry/api/v2/auth/DockerBearerTokenServiceSpec.groovy

```diff
@@ -33,7 +33,7 @@ class DockerBearerTokenServiceSpec extends Specification {
   DockerBearerTokenService tokenService
```

Comment:
Probably wouldn't hurt to have a test that verifies the behavior of the construct that expects a `passwordFile`.